libs/httpext/src/lib/http-ext.ts

```diff
@@ -72,8 +73,7 @@ export class HttpExt {
     plugin: Plugin;
   }): boolean {
     return (
-      typeof plugin.condition === 'function' &&
-      plugin.condition({ request }) === false
+      isFunction(plugin.condition) && plugin.condition({ request }) === false
```

Comment:
🤔 should we compare to:
- false
- !true
- truthy / falsy
- check if it's a boolean and throw an exception "please return a goddamn boolean!"

Reply:
IMO we should throw if it's not a boolean

Reply:
Return type check added